### PR TITLE
[GEOT-6324] WFS-NG online tests don't extend OnlineTestSupport

### PR DESCRIPTION
**Pull request by roarbra**
[GEOT-6324](https://osgeo-org.atlassian.net/browse/GEOT-6324) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=geotools&utm_campaign=badge_info)

Made AbstractWfsDataStoreOnlineTest subclass of OnlineTestSupport.

Added a fixture "wfs" to run those test cases that extended AbstractWfsDataStoreOnlineTest, and added an individual check for availability using HTTPClient instead on url.openStream().

Ignoring the test Capabilities200ServiceInfoOnlineTest because it didn't extend AbstractWfsDataStoreOnelineTest nor OnlineTestSupport, and because the url was wrong.

# Checklist

- [X] I have read the [contribution guidelines](https://github.com/geotools/geotools/blob/main/CONTRIBUTING.md).
- [X] I have sent a [Contribution Licence Agreement](https://docs.geotools.org/latest/developer/procedures/contribution_license.html) (not required for small changes, e.g., fixing typos in documentation).
- [X] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [X] Avoid [Java 9+ split packages](http://tutorials.jenkov.com/java/modules.html#split-packages-not-allowed).
- [x] All the build checks are green ([see automated QA checks](https://docs.geotools.org/latest/developer/conventions/code/qa.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [ ] [Documentation](https://github.com/geotools/geotools/tree/main/docs) has been updated (if change is visible to end users).
- [X] There is an issue in [GeoTools Jira](https://osgeo-org.atlassian.net/projects/GEOT) (except for changes not visible to end users). 
- [X] Commit message(s) must be in the form ``[GEOT-XYZW] Title of the Jira ticket``.
- [X] Bug fixes and small new features are presented as a single commit.
- [X] The commit targets a single objective (if multiple focuses cannot be avoided, each one is in its own commit, and has a separate ticket describing it).

### CODE REVIEW

File: modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java [hunk 1]
```diff
@@ -20,13 +20,15 @@
...
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Test class for {@link Capabilities200ServiceInfo}.
  *
  * @author Matthias Schulze (LDBV at ldbv dot bayern dot de)
  */
+@Ignore
```
Comment (mprins) [inline modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java:31]: This @Ignore  should at least have a reason  why it should be ignored eg. `@Ignore("service offline... or something")`

But I think you could update the `SERVER_URL` to https://laermkartierung.eisenbahn-bundesamt.de/deegree/services/wfs?service=WFS&request=GetCapabilities

**Review by mprins — CHANGES_REQUESTED** [review]

File: modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java (hunk 1 above)
Comment (roarbra) [reply modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java]: Tried that, but there are something with the certificate.

Comment (roarbra) [reply modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java]: @mprins Could you have a new look. I've changed to https, and added a reason for Ignore.
I also moved the test into "online" package.

Comment (aaime) [reply modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java]: @roarbra are you sure you tried the link mprins provided? The certificate is invalid for laermkartierung1.eisenbahn-bundesamt.de (as used in the test) but valid for laermkartierung.eisenbahn-bundesamt.de (notice the missing 1 in the name).

Comment (roarbra) [reply modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java]: Thanks, I didn't notice. Now it's working.

**Review by mprins — APPROVED** [review]

**Pull request merged**